fim_pipeline.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -e
+
+set -e
+
+# TODO
+# upgrade Dockerfile to add this as an env value
+projectDir=/foss_fim
+
+echo
+echo "======================= Start of fim_pipeline.sh ========================="
+echo "---- Started: `date -u`" 
+
```

Comment:
Should we include a `usage` function, or a description of what this script does at a high level?

Reply:
yes and no.  for fim_pipeline, it's usage and "help" is coming via fim_pre_processing.sh" as a proxy.

Reply:
I added a quick note above that comment line directing people to the fim_pre_processing.sh top